examples/multiplayer-netfox/scripts/player.gd

```diff
@@ -7,6 +7,7 @@ extends CharacterBody3D
 var gravity = ProjectSettings.get_setting("physics/3d/default_gravity")
 
 func _ready():
+	NetworkTime.on_tick.connect(_tick)
```

Comment:
For a while, the tick types were not differentiated, so everything was called `_tick`. The Player has a RollbackSynchronizer, so this method should be called `_rollback_tick`, with 3 params ( delta, tick, is_fresh ), and is called automatically without connecting to a signal.